packages/babel-helper-module-transforms/src/normalize-and-load-metadata.js

```diff
@@ -52,18 +54,7 @@ export type LocalExportMetadata = {
  * Check if the module has any exports that need handling.
  */
 export function hasExports(metadata: ModuleMetadata) {
```

Comment:
Is this function still needed?

Reply:
No, but as it is a separate package I cannot remove it without breaking backwards compatibility if a mix between old and new version of Babel packages is used. At first I also wanted to use meta.hasExports directly in the transforms, but I ended up not doing it for the same reason.